### PR TITLE
update readme & remove settings.json file

### PR DESCRIPTION
**Pull request by BigMorty**
## Purpose

* update readme & remove settings.json file

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[] Refactoring (no functional changes, no api changes)
[x] Documentation content changes
[x] Other... Please describe:
remove settings.json file from .vscode folder in case there are any specific settings

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code
<!-- Add steps to run the tests suite and/or manually test -->
```
See Readme for details
```

## What to Check
Verify that the following are valid
* Endpoints work

## Other Information
<!-- Add any other helpful information that may be needed here. -->

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,18 +55,19 @@ func start
...
 ```
 
 ### Using Visual Studio Code
 1) Open this folder in a new terminal
 2) Open VS Code by entering `code .` in the terminal
-3) Press Run/Debug (F5) to run in the debugger (select "Debug anyway" if prompted about local emulater not running) 
-4) Insure your favorite REST clientextension is installed (e.g. [RestClient in VS Code](https://marketplace.visualstudio.com/items?itemName=humao.rest-client), PostMan, etc.)
-5) Open the file src/functions/test/ which contains a GET and POST test
-6) Click the "Send Request" link for each and see the results in the right-hand pane that opens
+3) Make sure the Azure Functions extension is installed
```
Comment (mattchenderson) [inline README.md:67]:
```suggestion
3) Make sure the [Azure Functions extension](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurefunctions) is installed
```

File: README.md [hunk 2]
```diff
@@ -55,18 +55,19 @@ func start
...
+4) Add a **.vscode** folder by running *"Azure Functions: Initialize project for use with VS Code"* in the Command Pallete
```
Comment (mattchenderson) [inline README.md:68]: Same comment as other PR - might need to teach folks how to open this.

Comment (BigMorty) [reply README.md]: done

File: README.md [hunk 3]
```diff
@@ -55,18 +55,19 @@ func start
...
 
 ### Using Visual Studio Code
 1) Open this folder in a new terminal
 2) Open VS Code by entering `code .` in the terminal
-3) Press Run/Debug (F5) to run in the debugger (select "Debug anyway" if prompted about local emulater not running) 
-4) Insure your favorite REST clientextension is installed (e.g. [RestClient in VS Code](https://marketplace.visualstudio.com/items?itemName=humao.rest-client), PostMan, etc.)
-5) Open the file src/functions/test/ which contains a GET and POST test
-6) Click the "Send Request" link for each and see the results in the right-hand pane that opens
+3) Make sure the [Azure Functions extension](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurefunctions) is installed
+4) Add needed files to the **.vscode** folder by opening the command palette (`Crtl+Shift+P/Cmd+Shift+P` and selecting *"Azure Functions: Initialize project for use with VS Code"*
```
Comment (paulyuk) [inline README.md:68]: This is ok, but the following would be slightly clearer formatting if you like:
```suggestion
4) Add required files to the `.vscode` folder by opening the command palette using `Crtl+Shift+P` (or `Cmd+Shift+P` on Mac) and selecting *"Azure Functions: Initialize project for use with VS Code"*
```

**Review by paulyuk — APPROVED** [review]

**Pull request merged**